build-parent/pom.xml

```diff
@@ -575,8 +575,8 @@
                                         <build>
                                             <buildx>
                                                 <builderName>maven</builderName>
-                                                <cacheTo>type=local,dest=~/.docker/cache .</cacheTo>
-                                                <cacheFrom>type=local,src=~/.docker/cache .</cacheFrom>
+                                                <cacheTo>type=local,dest=${user.home}/.docker/cache</cacheTo>
```

Comment:
I: Currently on CI, the syntax above was creating a _literal_ directory named `~` instead of dereferencing to the home directory of the user. `$HOME` had a similar issue, so using the Java system property instead.  That way the expansion happens before the path is even passed to the fabric8 plugin.